clients/js/src/instructions/approvePluginAuthority.ts

```diff
@@ -1,39 +1,25 @@
 import { Context } from '@metaplex-foundation/umi';
 import { approvePluginAuthorityV1, PluginType } from '../generated';
-import {
-  isExternalPluginType,
-  PluginAuthority,
-  pluginAuthorityToBase,
-} from '../plugins';
-import { ExternalPluginKey } from '../plugins/externalPluginKey';
+import { PluginAuthority, pluginAuthorityToBase } from '../plugins';
+
+export type ApprovePluginAuthorityArgsPlugin = {
+  type: keyof typeof PluginType;
+};
 
 export type ApprovePluginAuthorityArgs = Omit<
   Parameters<typeof approvePluginAuthorityV1>[1],
   'pluginType' | 'newAuthority'
 > & {
-  plugin:
-    | {
-        type: keyof typeof PluginType;
-      }
-    | ExternalPluginKey;
+  plugin: ApprovePluginAuthorityArgsPlugin;
   newAuthority: PluginAuthority;
 };
 
 export const approvePluginAuthority = (
   context: Pick<Context, 'payer' | 'programs' | 'identity'>,
   { plugin, newAuthority, ...args }: ApprovePluginAuthorityArgs
-) => {
-  if (isExternalPluginType(plugin)) {
-    // TODO implement this
-    // return approveExternalPluginAuthorityV1(context, {
-    //   ...args,
-    //   key: externalPluginKeyToBase(plugin as ExternalPluginKey),
-    // });
-  }
-
-  return approvePluginAuthorityV1(context, {
```

Comment:
Not in scope for this PR, but for `updateDelegate` we have the currently unused `additinalDelegates` array, and I'm guessing we will be approving those delegates via an `updatePlugin` not `approvePluginAuthority` right?